independent-projects/arc/processor/src/main/java/io/quarkus/arc/processor/Methods.java

```diff
@@ -20,12 +20,19 @@
 import org.jboss.jandex.WildcardType;
 import org.jboss.logging.Logger;
 
+/**
+ * 
+ * @author Martin Kouba
+ * @author Michal Szynkiewicz, michal.l.szynkiewicz@gmail.com
+ */
```

Comment:
Didn't we decide to not add new `@author` tags?

Reply:
The deleted file (`MethodUtils`) had Michal as author and the functionality is now merged into this class, so the authors are just preserved.
Or is there a policy not to add `@author` at all to any files?

Reply:
Arc is still developed as an "independent" project and I'd like to keep this tag ;-).